components/Button/types.ts

```diff
@@ -0,0 +1,11 @@
+import { ButtonHTMLAttributes } from "react";
+
+export enum ButtonVariants {
```

Comment:
i suggest create directory enums and insert enums to this directory, because you import it from different places in your project